Assets/Project/Modules/Enemies/Slime/Scripts/SlimeRefactor/SlimeMediator.cs

```diff
@@ -145,6 +147,11 @@ public void Divide()
             else
             {
                 _slimeSounds.PlayDeathSound(_audioManager, _slimeTransform.gameObject, SlimeSizeID);
+                if (_explosive)
+                {
+                    Explosion explosion =  ServiceLocator.Instance.GetService<IHazardFactory>().CreateExplosion(_slimeTransform.position, Quaternion.identity, ExplosionSize.Small);
+                    explosion.StartExplosion();
+                }
```

Comment:
Lets not couple all the Slimes to the Explosion code.
I made a couple of scripts that will allow us to do this.
Currently it is in my branch, I'll show you how it works.